src/addons/attach/attach.js

```diff
@@ -11,12 +11,12 @@
     /*
      * CommonJS environment
      */
-    module.exports = attach(require('../../dist/xterm'));
+    module.exports = attach(require('../../xterm'));
   } else if (typeof define == 'function') {
     /*
      * Require.js is available
      */
-    define(['../../dist/xterm'], attach);
+    define(['../../xterm'], attach);
```

Comment:
Won't this break add-ons again in clients not compatible with ES6 modules? (I guess not, because they are moved into `src`)